packages-next/fields-document/src/DocumentEditor/component-blocks/index.tsx

```diff
@@ -326,21 +327,31 @@ function ComponentBlockRender({
   componentBlock,
   element,
   onElementChange,
-  children: _children,
+  children,
 }: {
   element: Element & { type: 'component-block' };
   onElementChange: (element: Partial<Element>) => void;
   componentBlock: ComponentBlock;
   children: any;
 }) {
   const childrenByPath: Record<string, ReactElement> = {};
-  const children = _children.type(_children.props).props.children;
```

Comment:
The children that Slate passes to element renderers is an array now so we don't have to do this hack now.